File: 백준/Bronze/1978. 소수 찾기/소수 찾기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int cntPrime = 0;
        int times = sc.nextInt();

        for (int i = 0; i < times; i++) {
            int cnt = 0;
            int num = sc.nextInt();

            if (num == 1) {
                continue;
            } else {
                for (int j = 1; j <= num; j++) {
                    if (num % j == 0) {
                        cnt++;
                    }
                }
            }

            if (cnt == 2) {
                cntPrime++;
            }
        }

        System.out.println(cntPrime);
    }
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.cc
#include <iostream>
using namespace std;

int main() {
	int times;
	cin >> times;

	int cntPrime = 0;

	for (int i = 0; i < times; i++)
	{
		int num;
		cin >> num;

		bool isPrime = true;

		if (num == 1)
		{
			isPrime = false;
		} 
		else {
			for (int j = 2; j <= num / 2; j++)
			{
				if (num % j == 0)
				{
					isPrime = false;
					break;
				}
			}
		}
		
		if (isPrime)
		{
			cntPrime++;
		}
	}

	cout << cntPrime;
}